tool/travis.sh

```diff
@@ -52,48 +52,36 @@ else
 
   echo ""
 
-  # Install dart_coveralls; gather and send coverage data.
-  if [ "$COVERALLS_TOKEN" ]; then
-
-    OBS_PORT=9292
+  OBS_PORT=9292
     
-    # Run the tests setup for coverage reporting.
-    dart --disable-service-auth-codes \
-      --disable-analytics \
-      --enable-vm-service=$OBS_PORT \
-      --pause-isolates-on-exit \
-      test/all.dart &
-
-    status=$?  
-
-    pub global activate coverage
-
-    echo "Collecting coverage on port $OBS_PORT..."
-
-    # Run the coverage collector to generate the JSON coverage report.
-    collect_coverage \
-      --port=$OBS_PORT \
-      --out=var/coverage.json \
-      --wait-paused \
-      --resume-isolates
-
-    echo "Generating LCOV report..."
-    format_coverage \
-      --lcov \
-      --in=var/coverage.json \
-      --out=var/lcov.info \
-      --packages=.packages \
-      --report-on=lib \
-      --check-ignore
-
-    echo "Running coveralls..."
-    gem install coveralls-lcov
-    coveralls-lcov var/lcov.info
+  # Run the tests setup for coverage reporting.
+  dart --disable-service-auth-codes \
+    --disable-analytics \
+    --enable-vm-service=$OBS_PORT \
+    --pause-isolates-on-exit \
+    test/all.dart &
```

Comment:
Now we just need to figure out how to do this IN the "real" test run. CC @grouma

Reply:
`package:test` supports a `--coverage` argument.